### PR TITLE
Improved ACL Token page

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
**Rebased on #3377**

Two big changes:

1. Only the token secret is necessary now
2. Authenticating a token lists its associated policies and rules

image

image

### CODE REVIEW

File: ui/app/controllers/settings/tokens.js [hunk 1]
```diff
@@ -1,20 +1,52 @@
...
+          this.setProperties({
+            tokenIsValid: true,
+            tokenIsInvalid: false,
+            tokenRecord: token,
+          });
+        },
+        () => {
+          this.set('token.secret', null);
+          this.setProperties({
+            tokenIsInvalid: true,
```
Comment (meirish) [inline ui/app/controllers/settings/tokens.js]: ha swapping the order here took me a second (before this it's always valid, invalid, record) - maybe swap it here for consistency?

File: ui/app/services/token.js [hunk 2]
```diff
@@ -4,20 +4,6 @@ import fetch from 'fetch';
 const { Service, computed, assign } = Ember;
 
 export default Service.extend({
-  accessor: computed({
```
Comment (meirish) [inline ui/app/services/token.js]: so with the move from accessor -> token, you'll just always do the lookup at the server now?

Comment (DingoEatingFuzz) [reply ui/app/services/token.js]: The lookup was always done on the server. Just now the accessor isn't necessary, due to a new `/acl/token/self` endpoint that looks for a token with a matching secret.

File: ui/app/controllers/settings/tokens.js (hunk 1 above)
Comment (DingoEatingFuzz) [reply ui/app/controllers/settings/tokens.js]: 👍 I dig it.

**Pull request merged**